src/main/java/org/intellij/sdk/codesync/utils/Queries.java

```diff
@@ -0,0 +1,47 @@
+package org.intellij.sdk.codesync.utils;
+
+public class Queries {
+
+    //TODO Turn this into multiline string.
+    public static final String CREATE_USER_TABLE = "CREATE TABLE IF NOT EXISTS user(EMAIL TEXT PRIMARY KEY, ACCESS_TOKEN TEXT, SECRET_KEY TEXT, ACCESS_KEY TEXT, IS_ACTIVE INT)";
+    public static class User{
+
+
+        public static final String TABLE_EXIST = Queries.table_exist("user");
+        public static String get_all(){
```

Comment:
Also seems like unused method, kindly confirm @syedgulahmed

Reply:
Yes, it is unused. I just wrote it we might need it in future.